packages/kit/src/views/Home/pages/HomePageView.tsx

```
@@ -23,16 +29,14 @@ import { WalletBackupAlert } from '../../../components/WalletBackup';
 import { HomeSupportedWallet } from '../components/HomeSupportedWallet';
-import useHomePageWidth from '../hooks/useHomePageWidth';
 
 import { HomeHeaderContainer } from './HomeHeaderContainer';
 import { NFTListContainerWithProvider } from './NFTListContainer';
 import { TabHeaderSettings } from './TabHeaderSettings';
 import { TokenListContainerWithProvider } from './TokenListContainer';
 import { TxHistoryListContainerWithProvider } from './TxHistoryContainer';
 import WalletContentWithAuth from './WalletContentWithAuth';
-
-let CONTENT_ITEM_WIDTH: Animated.Value | undefined;
+import { LayoutChangeEvent } from 'react-native';
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)